src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,44 @@
+import { AnchorHTMLAttributes, DetailedHTMLProps } from 'react';
+import classNames from 'classnames/bind';
+import { Link } from 'react-router-dom';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = DetailedHTMLProps<
```

Comment:
Can we add an `icon` property to be able to provide icons through `props` rather than `children`?

Reply:
Done